packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -577,7 +577,7 @@ export default class NotificationServicesController extends BaseController<
   #registerMessageHandlers(): void {
     this.messagingSystem.registerActionHandler(
       `${controllerName}:updateMetamaskNotificationsList`,
-      this.updateMetamaskNotificationsList.bind(this),
+      async (...args) => this.updateMetamaskNotificationsList(...args),
```

Comment:
I think this is okay, we can refactor if need be once we test on mobile.

Reason is, is because we have been burnt by Mobile Hermes not supporting async-arrow functions in the past.